app/pages/Dashboard.tsx

```diff
@@ -109,33 +108,55 @@ export default function Dashboard() {
         </div>
       </div>
 
-      <div className="mt-6 flex w-full flex-wrap px-10">
-        <div className="w-3/4">
-          <p className="mb-4 text-2xl text-black">Decentralized Identity Verification</p>
-          <p className="text-xl text-black">Select the verification stamps you’d like to connect to your Passport.</p>
-        </div>
+      <div>
+        {viewerConnection.status === "connecting" ? (
+          <div
```

Comment:
the "Waiting for wallet signature..." bar should be a little narrower in mobile view
![Figma mock on the left, Passport on the right](https://user-images.githubusercontent.com/62616399/177360707-a0cea63f-7b9c-4df8-a16a-eade2c93e218.png)